starfish/recipe/cli.py

```diff
@@ -0,0 +1,29 @@
+from slicedimage.io import resolve_path_or_url
+
+from starfish.config import StarfishConfig
+from starfish.starfish import starfish
+from starfish.util import click
+from .recipe import Recipe
+
+
+@starfish.command("recipe")
+@click.option("--recipe", required=True, type=str, metavar="RECIPE_PATH_OR_URL")
+@click.option(
+    "--input", type=str, multiple=True, metavar="INPUT_FILE_PATH_OR_URL",
+    help="input file paths or urls to map to the recipe input parameters")
+@click.option(
+    "--output", type=str, multiple=True, metavar="OUTPUT_FILE_PATH",
+    help="output file paths to write recipe outputs to")
+@click.pass_context
+def run_recipe(ctx, recipe, input, output):
```

Comment:
This needs usage examples. It's also not clear what any of the parameters must contain for the run to be valid. That should probably be documented here.

Reply:
It is recipe dependent.  Please let me know if #1245 is adequate.